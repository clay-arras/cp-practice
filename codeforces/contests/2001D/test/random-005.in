1
20
14 10 8 21 6 10 7 16 10 10 21 7 13 8 12 16 23 20 9 20
