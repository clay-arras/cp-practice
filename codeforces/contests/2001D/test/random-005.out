12
14 6 16 10 21 7 13 8 12 23 20 9
